pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -465,7 +485,8 @@ func buildSidecarListenerTLSContext(authenticationPolicy *authn.Policy) *auth.Do
 }
 
 // http specific listener options
-type httpListenerOpts struct { //nolint: maligned
+type httpListenerOpts struct {
+	//nolint: maligned
```

Comment:
`gofmt` should resolve alignment issues, right?  not sure which this being skipped with the `nolint` comment

Reply:
this is a struct alignment warning (linter wants us to reshuffle to fields to align the struct better). a fairly useless warning. gofmt isn't so "hands on" as to rearrange field order.